### ISSUE TITLE
Can't find file for current Linux GoG version of Hollow Knight

### ISSUE THREAD

**Issue by james-cnz** [fifty-six/Scarab#223]
### Scarab Version

v2.5.0.0

### Hollow Knight Version

v1.5.78.11833

### Steps to Reproduce

No special steps to reproduce, just normal installation and run.

### Expected Behaviour

I think "~/GOG Games/Hollow Knight/game/Hollow Knight" should be a recognised file?

### Actual Behaviour

Instead I think it seems to work if the file is renamed with the suffix .x86_64 .

### Logs

Probably not needed?

**Comment by fifty-six** [comment]
wdym recognized file?

**Comment by james-cnz** [comment]
The first time I ran Scarab, it asked for Hollow Knight's location.  It opened up a dialog box to pick a file, but no file in the Hollow Knight directories I looked in was shown to be picked.  I spent ages clicking through directories, and didn't find anything.  After I renamed "~/GOG Games/Hollow Knight/game/Hollow Knight" with the suffix .x86_64 , the file showed up, and I was able to pick it, and Scarab worked.
